### ISSUE TITLE
训练过程中没有任何错误提示的卡死

### ISSUE THREAD

**Issue by xf739645524** [MegEngine/MegEngine#188]
## 环境
1.系统环境：win10 + 1660ti + cuda 11.1 
2.MegEngine版本：  1.4.0
3.python版本： 3.7

## 复现步骤
连续训练两次, 都是在训练过程中卡死了 , 没有任何错误打印提示, cpu和内存都没有释放, gpu 内存还有很多余量, 内存也有大部分余量

## 请提供关键的代码片段便于追查问题
    a = UNetD(1)

    optimizer = mge.optimizer.Adam(a.parameters(), lr=0.001,)

    gm = autodiff.GradManager().attach(
        a.parameters(),
    )
 for step, (batch_data, batch_label) in tqdm(enumerate(data_loader)):

        data = mge.tensor(batch_data)
        label = mge.tensor(batch_label)

        with gm:

            logits = a(data)
            loss = Loss.l1_loss(logits, label)

            gm.backward(loss)
            optimizer.step().clear_grad()

## 请提供完整的日志及报错信息
405it [02:51,  2.36it/s]0.0010422633
0.001508836
409it [02:52,  2.37it/s]0.0014185604
411it [02:53,  2.37it/s]0.0012511967
0.0018083369
415it [02:55,  2.37it/s]0.0012596952
0.0021143884
419it [02:56,  2.37it/s]0.0018782918

**Comment by MegChai** [comment]
你好，问题已经解决了吗？有没有后续信息提供？

**Comment by xf739645524** [comment]
> 你好，问题已经解决了吗？有没有后续信息提供？

我之前以为是我代码的问题, 但是我修改之后发现还是会卡住, 不动, 没有任何错误提示

**Comment by MegChai** [comment]
> > 你好，问题已经解决了吗？有没有后续信息提供？
> 
> 另外请问下, 1.4.0的版本支持 cuda 11.1 吗? 我pip 安装是没有问题的

了解了，你可以参考一下这个问题： https://github.com/MegEngine/MegEngine/issues/151#issuecomment-776072789

MegEngine 目前发布的 Wheel 包并不支持 CUDA11+ 版本，所以会触发很长时间的在线编译。

在 https://github.com/MegEngine/MegEngine/issues/151 最新的回帖中有一个用户提供了 CUDA11+ 编译过的版本，可以试用。

**Comment by xf739645524** [comment]
> > > 你好，问题已经解决了吗？有没有后续信息提供？
> > 
> > 
> > 另外请问下, 1.4.0的版本支持 cuda 11.1 吗? 我pip 安装是没有问题的
> 
> 了解了，你可以参考一下这个问题： [#151 (comment)](https://github.com/MegEngine/MegEngine/issues/151#issuecomment-776072789)
> 
> MegEngine 目前发布的 Wheel 包并不支持 CUDA11+ 版本，所以会触发很长时间的在线编译。
> 
> 在 #151 最新的回帖中有一个用户提供了 CUDA11+ 编译过的版本，可以试用。

有windows的编译好的版本不?

**Comment by MegChai** [comment]
@xf739645524 没有，这种情况可能需要自己编译了，请参考 https://github.com/MegEngine/MegEngine#build

还有一种方法是使用 [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10), 但无法保证其有效性。

**Comment by xf739645524** [comment]
> @xf739645524 没有，这种情况可能需要自己编译了，请参考 https://github.com/MegEngine/MegEngine#build
> 
> 还有一种方法是使用 [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10), 但无法保证其有效性。

多谢答复, 我再看看是否有必要cuda版本降降级
